### PR TITLE
feat: logout docker registries in post step

### PR DESCRIPTION
**Pull request by clareliguori**
In the post-job cleanup, do a `docker logout` for all registries that had a successful `docker login` via this action.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cleanup.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            stderr: (data) => {
+              doLogoutStderr += data.toString();
+            }
+          }
+        });
+
+        if (exitCode != 0) {
+          core.debug(doLogoutStdout);
+          throw new Error('Could not logout: ' + doLogoutStderr);
+        }
```
[the remark this answers is not in the document]
Comment (clareliguori) [reply cleanup.js]: Yup that makes sense, fixed in latest commit

**Comment by rubenfonseca** [comment]
Due to this change, our CI pipeline broke. We have multiple workers on the same machine (same docker daemon) and now while they are pushing things to docker, as soon as one job finishes, it will wipe the credentials for the other jobs. Is there a way to make this behaviour optional?

**Comment by allisaurus** [comment]
@rubenfonseca  Thanks for reporting this, I'll cut and issue and we'll work on a fix.

**Comment by allisaurus** [comment]
@rubenfonseca in the meantime you can pin to a previous version of the action to get unblocked. See https://github.com/aws-actions/amazon-ecr-login/issues/77

**Comment by rubenfonseca** [comment]
That’s what we did and it worked. Thank you very much!

**Pull request merged**